optuna/_hypervolume/utils.py

```diff
@@ -1,5 +1,5 @@
 import numpy as np
-
+from optuna._hypervolume import WFG
```

Comment:
As mentioned by @hvy in this comment (https://github.com/optuna/optuna/pull/4223#discussion_r1038205294), I also think that this is the cause of the circular import. WFG requires `_compute_2d` and `_compute_2points_volume` in this file, but it requires `WFG`.

Alternatively, we may able to create a new file such as `optuna/_hypervolume/hssp.py` to define `_solve_hssp`. This is because
- we have some other solutions of HSSP, and
- WFG algorithm is implemented in a separate file (i.e., `optuna/_hypervolume/wfg.py`).

So, I suggested creating a new file for extensibility and consistency.

@not522 @hvy What do you think of it?

Reply:
Not necessarily a requirement in my opinion but sounds like a clean idea.